### ISSUE TITLE
Need a Committees index page with navigation

### ISSUE THREAD

**Issue by starksm64** [jakartaee/jakarta.ee#597]
**Is your feature request related to a problem? Please describe.**
There is no content tree for the steering, specification and marketing committee specific documents. 

**Describe the solution you'd like**
We should have a have a top level landing page that has a link from the Resource menu to a Committees page that lists the Steering, Specification, and Marketing Committees with associated subpages, and then have an index page for each committee that links to the relevant committee documents.

**Describe alternatives you've considered**
Previously content has existed in the committees specific github repository.

**Additional context**

The #594 issue to add the TCK process guide to the website leaves that content orphaned without any top level navigation or index page for the various committees.

**Comment by chrisguindon** [comment]
+1 for creating a new landing page.

Who is responsible for creating the initial content that we can use to create the page?

**Comment by chrisguindon** [comment]
Should we use the content from https://github.com/jakartaee/jakarta.ee/issues/593 to create the initial landing page? 

We could then add links to each committee at the top of the page?

**Comment by bshannon** [comment]
The document from #593 would be one of the items linked to from the landing page,
along with the committee-specific landing pages.

**Comment by autumnfound** [comment]
I've created a quick mockup of the page here: https://deploy-preview-599--jakartaee.netlify.com/committees/ 

The content was just a quick slapdash, but I wanted to get an idea down and pass it along. Something that might take more consideration is where will we put a link to this new top-level page, as our header is currently as full as we can get it with current styling/links.

![image](https://user-images.githubusercontent.com/23018241/72810824-60e3f080-3c2c-11ea-9434-684902b77992.png)

**Comment by bshannon** [comment]
I don't think that's quite what we want.

The top level "committees" page should just list the committees and give a brief description of their purpose.  It should link to a level level page, one for each
committee, where there will be more information about each committee, including
links to various document relevant to the committee - operations, process, meeting
schedule, meeting minutes, etc.

**Comment by starksm64** [comment]
I agree with @bshannon

**Comment by chrisguindon** [comment]
@autumnfound Thanks for your work on this so far!

I've unassigned you from this issue for now since I don't believe we are ready to implement this just yet.

We need to formulate a proposal based off feedback from this issue and then ask the community to give us feedback on our proposal. @TanjaObradovic is currently working on that!

Once this is done, we should be ready to execute on this task.

**Comment by waynebeaton** [comment]
How about we just add a "Committees" link to the main page that points to the [committees directory](https://jakarta.ee/committees/) for now. This would be a simple fix that gives us navigation to the content without having to put any effort into updating the index page when new content is added (Hugo will automatically update it). 

In parallel, we can work out and implement a more complete solution.

**Comment by TanjaObradovic** [comment]
As promised on the Specification Committee today we are expected to provide an update on how we plan to implement this and provide mockup. We can do as Martin suggested, we can add it to the About page, or under Resources drop-down, as Scott suggested. Whatever makes the most sense. @chrisguindon @shabnammayel

**Comment by chrisguindon** [comment]
## What is the request?

1. Create a hierarchy from jakarta.ee that will have the following structure
    - jakarta.ee/committees
    - jakarta.ee/committees/specification
    - jakarta.ee/committees/steering
    - jakarta.ee/committees/marketing

2. and visibly accessible from jakarta.ee.

### Proposed solution from EF webdev team
PR link: https://github.com/jakartaee/jakarta.ee/pull/604

1. Create a hierarchy from jakarta.ee that will have the following structure

    - https://deploy-preview-604--jakartaee.netlify.com/committees
    - https://deploy-preview-604--jakartaee.netlify.com/committees/specification
    - https://deploy-preview-604--jakartaee.netlify.com/committees/steering
    - https://deploy-preview-604--jakartaee.netlify.com/committees/marketing

2. and visibly accessible from jakarta.ee:
https://deploy-preview-604--jakartaee.netlify.com/about/

I added a learn more button below our list of committee members.

**Comment by chrisguindon** [comment]
> I don't think that's quite what we want.
> 
> The top level "committees" page should just list the committees and give a brief description of their purpose. It should link to a level level page, one for each
> committee, where there will be more information about each committee, including
> links to various document relevant to the committee - operations, process, meeting
> schedule, meeting minutes, etc.

I will need a description for each committee.

**Comment by chrisguindon** [comment]
> and visibly accessible from jakarta.ee:
> https://deploy-preview-604--jakartaee.netlify.com/about/
> 
> I added a learn more button below our list of committee members.

I can add a temporary Committee link under the top-level resource link as suggest by @starksm64 if my proposal is not enough!

Will someone step up and contribute content for these sections/pages? I don't think that I am best person to describe each of these committees :)

I would like us to focus on what we need to change to push this production!

My PR is not perfect but I am hopeful that we can make incremental improvements after this is live!

**Comment by TanjaObradovic** [comment]
I believe @chrisguindon proposal will work, however we do need a write-up for each page. We will get back to you regarding the text.
If any objections from the community - please let us know.

**Comment by shabnammayel** [comment]
Chris please use this document for the copy [Content for _committees (2).pdf](https://github.com/jakartaee/jakarta.ee/files/4103577/Content.for._committees.2.pdf)

**Comment by chrisguindon** [comment]
@shabnammayel Thanks for the content but it seems that the link from https://deploy-preview-604--jakartaee.netlify.com/committees to https://deploy-preview-604--jakartaee.netlify.com/committees/primer-on-committee-responsibilities.pdf has been drop in this pdf.

Is that on purpose?

**Comment by shabnammayel** [comment]
no it is not. and it's strange since I don't see that link in the pdf document that I attached when I download it.

**Comment by chrisguindon** [comment]
I added a description for each committee:
https://deploy-preview-604--jakartaee.netlify.com/committees/

@shabnammayel I added a link to the pdf on the word responsible.

Please advise if this is incorrect. The original request was made by @TanjaObradovic  via https://github.com/jakartaee/jakarta.ee/issues/593

I closed that issue and moved the conversation here since we are using this issue to create this new section!

**Comment by chrisguindon** [comment]
I created a new **Documents** section on the committee landing page for these generic documents:
https://deploy-preview-604--jakartaee.netlify.com/committees/

Please let me know if I need to change anything else!

**Comment by starksm64** [comment]
I think that looks pretty good. The only thing I don't see is navigation to the Committees page from the Resources pulldown menu,

**Comment by chrisguindon** [comment]
@starksm64 I added it now:
https://deploy-preview-604--jakartaee.netlify.com/committees/

**Comment by shabnammayel** [comment]
Please find the updated content. there;s been some minor changes
[Content for _committees (3).pdf](https://github.com/jakartaee/jakarta.ee/files/4105407/Content.for._committees.3.pdf)

**Comment by chrisguindon** [comment]
> Please find the updated content. there;s been some minor changes
> [Content for _committees (3).pdf](https://github.com/jakartaee/jakarta.ee/files/4105407/Content.for._committees.3.pdf)

Done: https://deploy-preview-604--jakartaee.netlify.com/committees/

Is there any more changes needed for this?

**Comment by starksm64** [comment]
I think that is good enough for the initial issue.

**Comment by chrisguindon** [comment]
> I think that is good enough for the initial issue.

Thanks! The page is now live :)
https://jakarta.ee/committees/

**Comment by TanjaObradovic** [comment]
@chrisguindon Many thanks!

**Comment by bshannon** [comment]
Sorry, I can't keep up with all this reviewing...

I think the lower level index pages should be titled (e.g.) "Specification Committee",
not just "Specifications".  The latter is way too ambiguous.  Ditto "Marketing Committee"
and "Steering Committee".

**Comment by chrisguindon** [comment]
I created a PR to add the word Committee in the title of each committee:
https://deploy-preview-611--jakartaee.netlify.com

**Comment by chrisguindon** [comment]
The commit was merged in! Closing this issue!

Please create a new issue for any new changes!
